webview/src/plots/components/customPlots/util.ts

```diff
@@ -0,0 +1,86 @@
+import { VisualizationSpec } from 'react-vega'
+
+export const createSpec = (metric: string, param: string) =>
+  ({
+    $schema: 'https://vega.github.io/schema/vega-lite/v5.json',
```

Comment:
Don't want to pin the y-axis to contain the 0 value:

<img width="1337" alt="image" src="https://user-images.githubusercontent.com/37993418/221739567-6032a296-c7bc-4793-b58b-5620a6e736db.png">

I think you want `scale: { zero: false },` in the y property.

Reply:
Great idea, thanks!